assets/shared/blocks/progress-bar/progress-bar.js

```diff
@@ -6,83 +6,65 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { sprintf, _n } from '@wordpress/i18n';
+import { sprintf, __ } from '@wordpress/i18n';
 
 /**
  * Course progress component.
  *
  * @param {Object}  props
- * @param {number}  props.lessonsCount            Number of lessons.
- * @param {number}  props.completedCount          Number of completed lessons.
+ * @param {number}  props.totalCount              Total count for progress bar.
+ * @param {number}  props.completedCount          Number of completed.
  * @param {boolean} props.hidePercentage          Hide completed percentage.
  * @param {Object}  props.wrapperAttributes       Wrapper HTML attributes.
  * @param {Object}  props.barWrapperAttributes    Bar wrapper HTML attributes.
- * @param {Object}  props.barAttributes           Bar HTML attributes.
  * @param {string}  props.countersClassName       Counters class name.
- * @param {string}  props.lessonsCountClassName   Lessons count class name.
  * @param {string}  props.completedCountClassName Completed count class name.
+ * @param {Object}  props.barAttributes           Bar HTML attributes.
  * @param {boolean} props.hideDefault             Hide default settings for edit view only.
+ * @param {boolean} props.progressBarLabel        Progress bar label.
  */
 const ProgressBar = ( {
-	lessonsCount,
+	totalCount,
 	completedCount,
 	hidePercentage,
 	wrapperAttributes,
 	barWrapperAttributes,
 	barAttributes,
 	countersClassName,
-	lessonsCountClassName,
 	completedCountClassName,
 	hideDefault,
+	progressBarLabel,
 } ) => {
 	const completePercentage =
-		Math.round( ( completedCount / lessonsCount ) * 100 ) || 0;
+		Math.round( ( completedCount / totalCount ) * 100 ) || 0;
 	const barPercentage = Math.max( hideDefault ? 0 : 3, completePercentage );
-
 	return (
 		<div { ...wrapperAttributes }>
 			<section
 				className={ classnames(
 					'sensei-progress-bar__heading',
+					'wp-block-sensei-lms-progress-heading',
```

Comment:
Just curious as to why we're adding these `wp-block` classes to various elements in this file? Are they needed?